extension/chrome/elements/pgp_block_modules/pgp-block-signature-module.ts

```diff
@@ -42,7 +43,9 @@ export class PgpBlockViewSignatureModule {
         } else {
           $('#pgp_signature').addClass('gray_label').text('verifying signature...');
           try {
-            const { pubkeys } = await this.view.pubLookup.lookupEmail(signerEmail);
+            const storedContact = await ContactStore.getOneWithAllPubkeys(undefined, signerEmail);
+            const shouldSkipOpenpgpOrg = storedContact?.sortedPubkeys.some(pubkey => pubkey.pubkey.usableForEncryption);
+            const { pubkeys } = await this.view.pubLookup.lookupEmail(signerEmail, shouldSkipOpenpgpOrg);
```

Comment:
This will later need some improving:

1) `usableForEncryption` is not a useful metric when dealing with signature. Should use `usableForSigning`
2) Reviewing the code, it seems we only run this when we already failed verification, not every time that message is verified. Is that true? I suppose we could fetch from `keys.openpgp.org` unconditionally here.

That means, all in all, I think these 3 lines can be reverted.